doop-central/data.go

```diff
@@ -0,0 +1,69 @@
+/*******************************************************************************
+*
+* Copyright 2021 SAP SE
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You should have received a copy of the License along with this
+* program. If not, you may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+*******************************************************************************/
+
+package main
+
+import (
+	"html/template"
+	"sort"
+)
+
+type data struct {
```

Comment:
`data` is quite generic. What about this?
```suggestion
type PreprocessedData struct {
```